rts/Game/Camera.cpp

```diff
@@ -22,6 +22,21 @@ CONFIG(float, EdgeMoveWidth)
 CONFIG(bool, EdgeMoveDynamic)
 	.defaultValue(true)
 	.description("If EdgeMove scrolling speed should fade with edge distance.");
+CONFIG(float, CameraMoveFastMult)
+	.defaultValue(3.0f)
+	.minimumValue(1.0f)
+	.description("The multiplier applied to speed when camera is in movefast state.");
+CONFIG(float, CameraMoveSlowMult)
+	.defaultValue(0.1f)
+	.maximumValue(1.0f)
+	.description("The multiplier applied to speed when camera is in moveslow state.");
+CONFIG(float, CameraFastScale)
+	.defaultValue(10.0f / 3.0f)
+	.description("Scaling for CameraMoveFastMult.");
+CONFIG(int, CamFrameTimeCorrection)
+    .defaultValue(0)
+	.minimumValue(0)
+	.description("Sets wether the camera interpolation factor should be the inverse of fps or last draw frame time (0 = lastdrawframetime, 1 = fpsinv)");
```

Comment:
This description doesn't sound useful. What are the practical differences? When would I use one or the other?

Reply:
I didn't write it, just moved it form here:
https://github.com/beyond-all-reason/spring/pull/475/files#diff-848d3fc035b7080bfa9e640de1240ae4b8db3fcc0c8031a27e56fc1848c8aba1L67
The relevant part of code where this is used is here
https://github.com/beyond-all-reason/spring/pull/475/files#diff-547482f9bca79127d00fb3fa6769a721b863db3e4c49c01f21ba1fa96fd4fad5R691-R692
It seems like useInterpolate changes whether the time difference between camera frames is calculated form last frame time or FPS. I am unsure how to improve this wording, but could give it a go.